modules/actor/actor-wfrp4e.js

```diff
@@ -1021,6 +1039,9 @@ export default class ActorWfrp4e extends Actor {
         testData.testDifficulty = game.wfrp4e.config.difficultyModifiers[html.find('[name="testDifficulty"]').val()];
         testData.successBonus = Number(html.find('[name="successBonus"]').val());
         testData.slBonus = Number(html.find('[name="slBonus"]').val());
+        if (testData.unofficialGrimoire) {
+          testData.ingredientMode = html.find('[name="ingredientTypeSelected"]').val();
+        }
```

Comment:
Can you add  `game.wfrp4e.utility.logHomebrew("unofficialgrimoire")`

Reply:
done